tests/tizen_capi/unittest_tizen_sensor.cc

```diff
@@ -18,6 +18,20 @@
 #include <gst/gst.h>
 #include <nnstreamer.h>
 #include <nnstreamer-capi-private.h>
+#include <unittest_util.h>
+
+static const uint TEST_TIME_OUT_TIZEN_SENSOR = 10000; /* timeout occur after 10 seconds */
```

Comment:
it's not 10 secs. `10000000` is 10 secs.

Reply:
wait_pipeline_process_buffers() accept in ms.
For better readability, TEST_TIME_OUT_TIZEN_SENSOR_MS would be better, though.

Reply:
oh... I realized it's changed to another unit at `wait_pipeline_process_buffers`. it's my mistake :)

Reply:
I will update variable name.
Apart from this,.. MS feels like mega..